telegram/_message.py

```diff
@@ -3303,31 +3308,29 @@ def _parse_html(
                             )
                             + insert
                         )
+                elif sys.maxunicode == max_value:
+                    html_text += message_text[last_offset : entity.offset - offset] + insert
```

Comment:
Just FYI: I don't think that we ever covered this in the CI

Reply:
do you want me to cover it? I think we'll have to monkeypatch it then.. I think we can just drop it since `sys.maxunicode` was only different for python 2 back in the day (https://docs.python.org/3/library/sys.html#sys.maxunicode). This code has been there since we had python 2 support, so do you want to just drop it?

Reply:
uhm, that might be possible, yes. Can you open a new issue for that?